src/AnalyticsAdapter.js

```diff
@@ -16,7 +16,8 @@ const {
     BID_ADJUSTMENT,
     BIDDER_DONE,
     SET_TARGETING,
-    AD_RENDER_FAILED
+    AD_RENDER_FAILED,
+    ADD_AD_UNITS
```

Comment:
Can you clarify on the changes in this file?  

If a new analytics event is needed, that should be part of a separate PR (since it's a core change to the module, rather than just your adapter).

Reply:
Sure, 

The event `addAdUnits` helps us detect any change on the adUnits object between the moment a new value is assigned to it and any other moment in the flow.

We are currently handling it at a lower level with:

```    
    pbjsGlobal.onEvent('addAdUnits', (data) => { myHandler(data); });
```

but I understood that the events we get from it are not considered a stable interface.

I will create a separate PR fo it.

Reply:
@jsnellbaker I updated this PR. Now it is only about rivrAnalyticsAdapter.

Reply:
@jsnellbaker  PR for changes in core have been moved to #3691